### DISCUSSION TITLE
It seems the setup guide is down

### DISCUSSION THREAD

**Discussion by nikepupu** [coder/code-server#4057] [General] [answered]
https://coder.com/docs/code-server/latest/guide

**Comment by jsjoeio** [comment]
Hmm...not sure what could have broke 🤔 Thanks for letting us know. We will take a look on Monday!

**Reply by lg-nineoneninethreenine** [reply]
The entire guide page is down! The homepage is totally fine, because when I hit Go Home, it does work. In the recent PR, that Vercel message that showed a demo also failed, it didn't work. Check Vercel, it might have answers.

**Reply by jsjoeio** [reply]
Yeah, something went wrong in our PR. I'm going to take a closer look and fix

**Comment by jsjoeio** [comment, marked as answer]
It's back up now! Sorry for the trouble.

https://coder.com/docs/code-server/latest
